translations/checker/index.ts

```diff
@@ -0,0 +1,82 @@
+import * as findInFiles from 'find-in-files'
+import { uniq } from 'lodash'
+import { translations } from '../translations'
+import * as fs from 'fs'
+
+interface MissingLanguageItem {
+  totalCount: number
+  missingCount: number
+  labels: {
+    [key: string]: string[]
+  }
+}
+
+interface MissingLanguage {
+  [key: string]: MissingLanguageItem
+}
+
+const getAllTranslationsKeys = (keys: string[], map: Map<string, string[]>): Map<string, string[]> => {
+  keys.forEach((k) => {
+    let item = k.replace('translate(', '').replace(')', '').split(',')
+    item = item.map((v) => v.trim().replace(/^'(.*)'$/, '$1'))
+    if ((item[0] !== '' && item[0] != null) && (item[1] !== '' && item[1] != null)) {
+      const list = map.get(item[0]) ?? []
+      list.push(item[1])
+      map.set(item[0], uniq(list))
+    }
+  })
+  return map
+}
+
+const updateBaseTranslations = async (baseTranslation: Map<string, string[]>): Promise<Map<string, string[]>> => {
+  const translationRegex = new RegExp(/translate\(.*,+.*\)/)
+  const find = async (directory: string): Promise<findInFiles.FindResult> => await findInFiles.find(translationRegex, directory, '.tsx$')
+  // list of folders to check
+  const directories = ['components', 'navigation', 'screens']
+  for (const dir of directories) {
+    const result = await find(dir)
+    Object.keys(result).forEach((k) => {
+      baseTranslation = getAllTranslationsKeys(uniq(result[k].matches), baseTranslation)
+    })
+  }
+  return baseTranslation
+}
+
+const checkTranslations = (baseTranslation: Map<string, string[]>, missingTranslations: MissingLanguage): MissingLanguage => {
+  const languages = Object.keys(translations)
+  let totalCount = 0
+  baseTranslation.forEach((labels, screenName) => {
+    totalCount = totalCount + labels.length
+    languages.forEach((language) => {
+      const langItem: any = { ...translations }[language]
+      const languageTranslations = missingTranslations[language] ?? { missingCount: 0, labels: {}, totalCount }
+      const translationFile = langItem[screenName]
+      if (translationFile == null) {
+        languageTranslations.labels[screenName] = labels
+        languageTranslations.missingCount = languageTranslations.missingCount + labels.length
+      } else {
+        labels.forEach((baseLabel) => {
+          if (translationFile[baseLabel] == null) {
+            languageTranslations.labels[screenName] = languageTranslations.labels[screenName] ?? []
+            languageTranslations.labels[screenName].push(baseLabel)
+            languageTranslations.missingCount = languageTranslations.missingCount + 1
+          }
+        })
+      }
+      languageTranslations.totalCount = totalCount
+      missingTranslations[language] = languageTranslations
+    })
+  })
+  fs.writeFileSync('./missingTranslations.json', JSON.stringify(missingTranslations, null, 4))
```

Comment:
```suggestion
  fs.writeFileSync('./missing_translations.json', JSON.stringify(missingTranslations, null, 4))
```

For naming consistency.